script/modules_collector.py

```diff
@@ -0,0 +1,61 @@
+import os
```

Comment:
AFAIK we are trying to get rid of Python scripts. I would prefer this script to be written in Golang.

Reply:
@kvch this script is not new, it has been moved from `metricbeat/scripts`, I am +1 on rewritting it in Golang, but I think this could be also done in other PR.

Reply:
Ah, I wasn't aware that we were trying to convert our build scripts to Golang. Makes sense and TIL!

So I'm also +1 for porting over this script but in a follow up PR, since this PR _mostly_ moves the existing script around to use it from filebeat as well as metricbeat. @kvch are you okay if I do this work in a follow up PR?